wrench/src/main.rs

```diff
@@ -597,6 +597,7 @@ fn main() {
             None
         } else {
             let surface = instance.create_surface(window.get_window().unwrap());
+            dim = window.get_inner_size();
```

Comment:
isn't it already guaranteed to be `window.get_inner_size()` at this point? or do you see the size changing when a surface is created?

Reply:
Yes, exactly. Sometimes we receive invalid window size before the surface creation. After it created we always got the valid window size.